cypress/support/pages/tasks/pageTasks.js

```diff
@@ -24,11 +49,11 @@ class Tasks {
   }
 
   fileUploadButtonIsDisabled() {
-    cy.get(Tasks.#fileUploadButton).find('div').should('have.class', 'form-files-storage-disabled')
+    cy.get(Tasks.#fileUploadButtonDisabled).should('be.visible')
   }
 
   fileUploadButtonIsEnabled() {
```

Comment:
same here file name can be -> uploadFileButtonEnabled

Reply:
changed to seeUploadFileButtonIsDisabled